client/src/pages/Register.js

```diff
@@ -62,10 +62,17 @@ const Register = () => {
       .then(res => {
         if (res.data.success) {
           localStorage.setItem("email", res.data.user.username);
+
           history.push("/settings");
         }
       })
+
       .catch(() => setRegisterError("Sorry, this email already exists."));
+    axios.put(
+      `/settings/${localStorage.getItem("email")}/company/${
+        event.target.company.value
+      }`
+    );
```

Comment:
Do we need this since the company would be added to the user settings on the register post route?

Reply:
good point, we don't need it. I updated it.